src/tokens/solana.tokenlist.json

```diff
@@ -28751,4 +28767,4 @@
     "minor": 3,
     "patch": 3
   }
-}
\ No newline at end of file
+}
```

Comment:
This is what is causing your auto merge error btw.

Reply:
thanks, this is fixed.

Reply:
Do I need to submit a new PR, or I just wait for 1 hour and it should be fine?